src/analysis.jl

```diff
@@ -130,3 +130,10 @@ function potential_energy(s::Simulation)
 
     return uconvert(s.energy_unit, potential)
 end
+
+function vacf(vl::VelocityLogger)
+	n_atoms = length(first(vl.velocities))
+	
+	return (1/n_atoms)*(dot(first(vl.velocities), last(vl.velocities)))
```

Comment:
Might be worth having arguments to the function to specify the starting and ending indices, e.g. `vacf(vl, 10, 20)` would do `dot(vl.velocities[10], vl.velocities[20])`. The defaults could be the first and last frame, i.e. `1` and `length(vl.velocities)`, so `vacf(vl)` gives the above behaviour.

Reply:
Done. I changed the name to velocityautocorr too (I made it vacf at first since that's the compute command in LAMMPS), this is a much better name.